pycalphad/core/eqsolver.pyx

```diff
@@ -162,14 +162,26 @@ cdef bint add_new_phases(object composition_sets, object removed_compsets, objec
         return True
     return False
 
-cpdef pointsolve(composition_sets, comps, cur_conds, problem, iter_solver):
-    "Mutates composititon_sets with updated values if it converges. Returns SolverResult."
-    return _solve_and_update_if_converged(composition_sets, comps, cur_conds, problem, iter_solver)
+cpdef globalmin_solve(composition_sets, comps, cur_conds, prob, state_variables, sample_points, iter_solver):
```

Comment:
Maybe @richardotis can correct me, but I believe a `cpdef` should be equivalent in performance to a `cdef`. See this [StackOverflow post](https://stackoverflow.com/a/49172529)